inc/modules/export/epub/class-epub201.php

```diff
@@ -2314,6 +2314,8 @@ protected function fuzzyHrefMatch( $url, $pos ) {
 			return false;
 		}
 
+		$order = $this->fixOrder( $lookup['__order'] );
```

Comment:
Note: Will fix on  every call to `fuzzyHrefMatch` which could be many.

Do this only once?

Reply:
Suggestion.

```
		static $lookup = false; // Cheap cache
		static $order = false;
		if ( false === $lookup && $order === false ) {
			$lookup = Book::getBookStructure();
			$order = $this->fixOrder( $lookup['__order'] );
		}
```